frontend/package.json

```diff
@@ -6,27 +6,61 @@
   "description": "Polkadot Telemetry frontend",
   "scripts": {
     "precommit": "lint-staged",
-    "start": "react-scripts-ts start",
-    "build": "react-scripts-ts build",
+    "start": "node scripts/start.js",
+    "build": "node scripts/build.js",
     "test": "TS_NODE_PROJECT=tsconfig.test.json tape -r ts-node/register test/**/*spec.ts",
     "test:coverage": "nyc yarn test",
-    "eject": "react-scripts-ts eject",
     "pretty:check": "prettier --check src/**/*.{ts,tsx}",
     "pretty:fix": "prettier --write src",
     "clean": "rm -rf node_modules build .nyc ./tmp/env-config.js report*.json yarn-error.log"
   },
   "dependencies": {
     "@polkadot/util-crypto": "^2.8.1",
     "@types/react-measure": "^2.0.6",
+    "autoprefixer": "7.1.6",
+    "babel-jest": "20.0.3",
+    "babel-loader": "7.1.2",
+    "babel-preset-react-app": "^3.1.2",
     "blakejs": "^1.1.0",
+    "case-sensitive-paths-webpack-plugin": "2.1.1",
+    "chalk": "1.1.3",
+    "css-loader": "0.28.7",
+    "dotenv": "4.0.0",
+    "dotenv-expand": "4.2.0",
+    "extract-text-webpack-plugin": "3.0.2",
+    "file-loader": "1.1.5",
+    "fork-ts-checker-webpack-plugin": "^0.2.8",
+    "fs-extra": "3.0.1",
+    "html-webpack-plugin": "2.29.0",
     "husky": "^4.2.5",
+    "jest": "20.0.4",
     "lint-staged": "^10.1.7",
+    "object-assign": "4.1.1",
+    "postcss-flexbugs-fixes": "3.2.0",
+    "postcss-loader": "2.0.8",
+    "promise": "8.0.1",
+    "raf": "3.4.0",
     "react": "^17.0.1",
+    "react-dev-utils": "^5.0.2",
     "react-dom": "^17.0.1",
     "react-measure": "^2.3.0",
-    "react-scripts-ts": "3.1.0",
+    "resolve": "1.6.0",
+    "source-map-loader": "^0.2.1",
     "stable": "^0.1.8",
-    "tslint": "^6.1.1"
+    "style-loader": "0.19.0",
+    "sw-precache-webpack-plugin": "0.11.4",
+    "ts-jest": "22.0.1",
+    "ts-loader": "^2.3.7",
+    "tsconfig-paths-webpack-plugin": "^2.0.0",
+    "tslint": "^5.7.0",
+    "tslint-config-prettier": "^1.10.0",
+    "tslint-react": "^3.2.0",
+    "uglifyjs-webpack-plugin": "1.2.5",
+    "url-loader": "0.6.2",
+    "webpack": "3.8.1",
+    "webpack-dev-server": "2.11.3",
+    "webpack-manifest-plugin": "1.3.2",
+    "whatwg-fetch": "2.0.3"
```

Comment:
How much of these are actually devDependencies?

(side note; do we need the polyfills like `whatwg-fetch` nowadays? I'd have assumed not)